src/assessments/common/test-automatically-pass-notice.tsx

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import * as React from 'react';
+import * as Markup from '../markup';
+
+const TestAutomaticallyPassedNotice = (
+    <p>
+        <Markup.Emphasis>
+            Note: If no matching/failing instances are found, this requirement will automatically be marked as pass.
+        </Markup.Emphasis>
+    </p>
+);
+
+export default TestAutomaticallyPassedNotice;
```

Comment:
`export default` should throw a tslint error here (on vscode). This is part of the tslint rules we want to enforce so, can you do a normal export? This means you need to update all the imports of this component

Reply:
Agree with that on the export. Also, this should be a NamedSFC, not an simple const. This way it can render as an element. Specify as `<TestAutomaticallyPassNotice>`.

Also, this should be in content.

Reply:
Maybe move it to `src/content/test/common/test-automatically-pass-notice.tsx`

Reply:
Accepted